radixdlt-engine/src/main/java/com/radixdlt/atommodel/tokens/UnallocatedTokensParticle.java

```diff
@@ -75,6 +75,21 @@ public UnallocatedTokensParticle(
 		this.tokenPermissions = ImmutableMap.copyOf(tokenPermissions);
 	}
 
+	public UnallocatedTokensParticle(
+			UInt256 amount,
+			UInt256 granularity,
+			RRI tokenDefinitionReference,
+			Map<TokenTransition, TokenPermission> tokenPermissions,
+			long nonce
+	) {
```

Comment:
Something wrong with indentation. Should be single tab from the start of `public` keyword.